man/systemd.unit.xml

```diff
@@ -1975,6 +1975,11 @@
             <entry>Cache directory root</entry>
             <entry>This is either <filename>/var/cache</filename> (for the system manager) or the path <literal>$XDG_CACHE_HOME</literal> resolves to (for user managers).</entry>
           </row>
+          <row>
+            <entry><literal>%d</literal></entry>
+            <entry>Credentials directory</entry>
+            <entry>This is the value of the <literal>$CREDENTIALS_DIRECTORY</literal> environment variable if available. See section "Credentials" in <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry> for more information.</entry>
```

Comment:
that's not quite the case. Its the directory *root*, i.e. the unit name has to be suffixed to get the per-unit dir.

Reply:
Is it? We explicitly concatenate it with the unit name in both cases iiuc:

https://github.com/systemd/systemd/blob/c8a4306e63a5387813e6e01164e5303e28dc60dc/src/core/execute.c#L1984

Reply:
yes, that's what I am saying: `%d` should map to `p->prefix[EXEC_DIRECTORY_RUNTIME]` suffixed with `/credentials`, but *without* the `u->id` additional suffix.

Thus, i'd call this a "root" directory, because in it you find one subdir for each service.

Reply:
or hmm, maybe the way you defined it actually does make more sense.